# POIInfo.msg
string name
string type
string description
geometry_msgs/Point position
poi_state_server/NamedPoint[] candidate_points
geometry_msgs/Point boundary_min
geometry_msgs/Point boundary_max
bool has_boundary

